File: app/src/main/java/uz/murodjon_sattorov/firebasechatapp/adapters/pageradapter/SignInUpAdapter.java
package uz.murodjon_sattorov.firebasechatapp.adapters.pageradapter;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

import java.util.ArrayList;

/**
 * Created by <a href="mailto: sattorovmurodjon43@gmail.com">Murodjon Sattorov</a>
 *
 * @author Murodjon
 * @date 9/2/2021
 * @project Firebase Chat app
 */
public class SignInUpAdapter extends FragmentPagerAdapter {

    ArrayList<Fragment> fragmentArrayList = new ArrayList<>();
    ArrayList<String> stringArrayList = new ArrayList<>();

    public SignInUpAdapter(@NonNull FragmentManager fm) {
        super(fm);
    }

    @NonNull
    @Override
    public Fragment getItem(int position) {
        return fragmentArrayList.get(position);
    }

    @Override
    public int getCount() {
        return stringArrayList.size();
    }

    public void addPagerFragment(Fragment fragment, String s) {
        fragmentArrayList.add(fragment);
        stringArrayList.add(s);
    }

    @Nullable
    @Override
    public CharSequence getPageTitle(int position) {
        return stringArrayList.get(position);
    }

}

File: app/src/main/java/uz/murodjon_sattorov/firebasechatapp/adapters/pageradapter/SignInUpAdapter.kt
package uz.murodjon_sattorov.firebasechatapp.adapters.pageradapter

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter
import java.util.*

/**
 * Created by [Murodjon Sattorov](mailto: sattorovmurodjon43@gmail.com)
 *
 * @author Murodjon
 * @date 9/2/2021
 * @project Firebase Chat app
 */
class SignInUpAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
    var fragmentArrayList = ArrayList<Fragment>()
    var stringArrayList = ArrayList<String>()
    override fun getItem(position: Int): Fragment {
        return fragmentArrayList[position]
    }

    override fun getCount(): Int {
        return stringArrayList.size
    }

    fun addPagerFragment(fragment: Fragment, s: String) {
        fragmentArrayList.add(fragment)
        stringArrayList.add(s)
    }

    override fun getPageTitle(position: Int): CharSequence? {
        return stringArrayList[position]
    }
}